internal/provisioner/cluster.go

```diff
@@ -84,30 +87,49 @@ func CreateCluster(provider, size string, logger log.FieldLogger) error {
 func DeleteCluster(clusterId string, logger log.FieldLogger) error {
 	logger = logger.WithField("cluster", clusterId)
 
-	logger.Info("deleting cluster")
-
 	s3StateStore := "dev.cloud.mattermost.com"
 	dns := fmt.Sprintf("%s-kops.k8s.local", clusterId)
 
-	// Temporarily look for the kops output directory as a local folder named `tmp`. See above.
-	outputDir := "tmp"
+	// Temporarily look for the kops output directory as a local folder named after
+	// the cluster ID. See above.
+	outputDir := fmt.Sprintf("cluster-%s", clusterId)
+
+	// Validate the provided cluster ID before we alter state in any way.
+	logger.Info("verifying cluster ID")
```

Comment:
nit: This is going to basically fail immediately, so is the log that useful? To me, this falls under the same class of errors as the input checks in `CreateCluster` above, which don't need "starting logs".